### DISCUSSION TITLE
希望能开一期平台调用，pinvoke源生成(libraryimport)，com互操作的一节课。

### DISCUSSION THREAD

**Discussion by Gaoyifei1011** [BYJRK/DotNet-Discussions#91] [C# 技术交流]
希望能开一期平台调用，pinvoke源生成(libraryimport)，com互操作的一节课。

我的项目有时候涉及到了平台调用(源生成)，com互操作的一些知识，但是没有好的学习材料，所以希望能从您这里学习一下

**Comment by Dynesshely** [comment]
个人认为 MSDN 上的文档解释得足够清楚

https://learn.microsoft.com/en-us/dotnet/standard/native-interop/pinvoke

若您想查看中文版本, 可以将 url 中的 `en-us` 替换为 `zh-cn`
